openlcsd/flow/tasks.py

```diff
@@ -1118,37 +1205,73 @@ def fork_remote_source_components_imports(context, engine, rs_comps, src_dir):
     engine.logger.info('Done')
 
 
-def fork_components_imports(context, engine):
+def fork_components_imports(context, engine, parent, components):
+    """
+    Fork tasks for components import
+    """
+    cli = context.get('client')
+    url = '/sources/import/'
+    data = {
+        'components': components,
+        'license_scan': context.get('license_scan', True),
+        'copyright_scan': context.get('copyright_scan', True),
+        'parent_task_id': context.get('task_id'),
+        'token': encrypt_with_secret_key(
+            cli.headers['Authorization'].split()[-1],
+            context['config']['TOKEN_SECRET_KEY']
+        ),
+        'token_sk': context['config']['TOKEN_SECRET_KEY'],
+        'provenance': context.get('provenance')
```

Comment:
I'm not sure I get it here. You already passed in `components`, isn't it always from corgi? why bother use `context.get("provenance")? Or am I missing anything?